### PR TITLE
RavenDB-22271 - Audit log on revisions settings changes (V6.0)

### PR DESCRIPTION
**Pull request by shaharhikri**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-22271/Audit-log-on-revisions-settings-changes

### Additional description

Audit log on revisions settings changes

### Type of change

- [ ] Bug fix
- [ ] Regression bug fix
- [ ] Optimization
- [x] New feature

### How risky is the change?

- [x] Low 
- [ ] Moderate 
- [ ] High
- [ ] Not relevant

### Backward compatibility

- [x] Non breaking change
- [ ] Ensured. Please explain how has it been implemented?
- [ ] Breaking change
- [ ] Not relevant

### Is it platform specific issue?

- [ ] Yes. Please list the affected platforms.
- [x] No

### Documentation update

- [ ] This change requires a documentation update. Please mark the issue on YouTrack using `Documentation Required` tag.
- [x] No documentation update is needed 

### Testing by Contributor

- [ ] Tests have been added that prove the fix is effective or that the feature works
- [ ] Internal classes added to the test class (e.g. entity or index definition classes) have the lowest possible access modifier (preferable `private`) 
- [x] It has been verified by manual testing

### Testing by RavenDB QA team

- [ ] This change requires a special QA testing due to possible performance or resources usage implications (CPU, memory, IO). Please mark the issue on YouTrack using `QA Required` tag.
- [x] No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?

- [ ] Yes. Please list the affected features/subsystems and provide appropriate explanation
- [x] No

### UI work

- [ ] It requires further work in the Studio. Please mark the issue on YouTrack using `Studio Required` tag.
- [x] No UI work is needed

### CODE REVIEW

File: src/Raven.Server/Documents/Handlers/Admin/AdminRevisionsHandler.cs [hunk 1]
```diff
@@ -20,13 +20,17 @@ public async Task DeleteRevisionsFor()
...
+        public const string ConflictedRevisionsConfigTag = "conflicted-revisions-config";
+
         [RavenAction("/databases/*/admin/revisions/conflicts/config", "POST", AuthorizationStatus.DatabaseAdmin)]
         public async Task ConfigConflictedRevisions()
         {
             using (var processor = new AdminRevisionsHandlerProcessorForPostRevisionsConflictsConfiguration(this))
                 await processor.ExecuteAsync();
         }
 
+        public const string ReadRevisionsConfigTag = "read-revisions-config";
```
Comment (karmeli87) [inline src/Raven.Server/Documents/Handlers/Admin/AdminRevisionsHandler.cs]: why is that here? let's be consistent

File: src/Raven.Server/Documents/Handlers/Admin/AdminRevisionsHandler.cs [hunk 2]
```diff
@@ -20,13 +20,17 @@ public async Task DeleteRevisionsFor()
                 await processor.ExecuteAsync();
         }
 
+        public const string ConflictedRevisionsConfigTag = "conflicted-revisions-config";
```
Comment (karmeli87) [inline src/Raven.Server/Documents/Handlers/Admin/AdminRevisionsHandler.cs]: why is that here? let's be consistent

File: src/Raven.Server/Documents/Handlers/Processors/Databases/AbstractHandlerProcessorForUpdateDatabaseConfiguration.cs [hunk 3]
```diff
@@ -17,6 +17,7 @@ internal abstract class AbstractHandlerProcessorForUpdateDatabaseConfiguration<T
     where T : class 
 {
     private readonly bool _isBlittable;
+    protected long _index;
```
Comment (karmeli87) [inline src/Raven.Server/Documents/Handlers/Processors/Databases/AbstractHandlerProcessorForUpdateDatabaseConfiguration.cs]: `_index` -> `Index`

File: src/Raven.Server/Documents/Handlers/Admin/AdminRevisionsHandler.cs (hunk 1 above)
Comment (shaharhikri) [reply src/Raven.Server/Documents/Handlers/Admin/AdminRevisionsHandler.cs]: Done

File: src/Raven.Server/Documents/Handlers/Admin/AdminRevisionsHandler.cs (hunk 2 above)
Comment (shaharhikri) [reply src/Raven.Server/Documents/Handlers/Admin/AdminRevisionsHandler.cs]: Done

File: src/Raven.Server/Documents/Handlers/Processors/Databases/AbstractHandlerProcessorForUpdateDatabaseConfiguration.cs (hunk 3 above)
Comment (shaharhikri) [reply src/Raven.Server/Documents/Handlers/Processors/Databases/AbstractHandlerProcessorForUpdateDatabaseConfiguration.cs]: Done

**Review by karmeli87 — APPROVED** [review]
